### PR TITLE
add example projection, fix other examples

### PR DESCRIPTION
**Pull request by matbesancon**
Adding example with a rotating circle and its projection onto the positive orthant.

Based on:
https://matbesancon.github.io/post/2020-12-24-chains_sets2/

I also fixed the other examples which did not accommodate running them from another folder, relative paths should use `@__DIR__`.

Also, running Javis often leads to an error when ImageIO and ImageMagick are not installed, I added these

### CODE REVIEW

**Comment by matbesancon** [comment]
weird, I did run the formatter with the blue code style on the only added file

File: examples/follow_path.jl [hunk 1]
```diff
@@ -2,12 +2,12 @@ using Javis, Animations
 
 function ground(args...)
     background("white")
-    sethue("black")
+    return sethue("black")
```
Comment (Wikunia) [inline examples/follow_path.jl:5]: Why did you add `return` here?

**Comment by Wikunia** [comment]
Thanks for the example!
Hmm seems like the formatter isn't happy yet. Did you run `using JuliaFormatter; format(".")` in the root folder?
Do you mind adding your example to https://github.com/Wikunia/Javis.jl/blob/master/docs/src/examples.md ?

**Comment by Wikunia** [comment]
@TheCedarPrince we should figure out why ImageIO and ImageMagick were needed for @matbesancon .

**Comment by matbesancon** [comment]
>  Hmm seems like the formatter isn't happy yet. Did you run using JuliaFormatter; format(".") in the root folder?

I ran it on the examples only. Let me check if I run on root

**Comment by matbesancon** [comment]
I just ran JuliaFormatter on everything, it seems some files were not compliant

**Comment by matbesancon** [comment]
Here is the result I get by formatting the file the action complains about:
image

**Comment by matbesancon** [comment]
@Wikunia there is a problem with the javis_viewer.jl file, basically the Formatter can run multiple times and always finds something to change

**Comment by Wikunia** [comment]
I made a PR for your fork @matbesancon . When I rerun formatter locally it says that it's already formatted. It behaved weird here such that I changed the comment a bit (removed newlines)

**Comment by matbesancon** [comment]
Yep just saw and merged it thanks

File: examples/follow_path.jl (hunk 1 above)
Comment (Wikunia) [reply examples/follow_path.jl:5]: I prefer `sethue("black")` without the `return` as it's not needed afterwars.

Comment (matbesancon) [reply examples/follow_path.jl:5]: it was added by the formatter

Comment (matbesancon) [reply examples/follow_path.jl:5]: if you don't want to set returns at the bottom of all functions, is should be specified in the Formatter TOML I believe

Comment (matbesancon) [reply examples/follow_path.jl:5]: also, see:
https://github.com/invenia/BlueStyle#method-definitions
it is advised by the blue style, that's why it's there by default

**Comment by Wikunia** [comment]
Interesting as it wasn't there before your change

**Comment by matbesancon** [comment]
did you try running formatter on everything? That's when it happened

**Comment by Wikunia** [comment]
I always run it on root. I think I'll change the config for it. First we (@TheCedarPrince and I) should figure out why InageMagick is needed.

**Comment by Wikunia** [comment]
Alright. That took much longer than hoped but looks good to me. I'll merge this and change the return in an upcoming PR where I'll fix the configuration for the formatter as well.
